src/Standards/Squiz/Sniffs/WhiteSpace/OperatorSpacingSniff.php

```diff
@@ -57,6 +64,45 @@ public function register()
         $targets[] = T_INLINE_ELSE;
         $targets[] = T_INSTANCEOF;
 
+        $this->nonOperandTokens = array_merge(
+            // Trying to operate on a negative value; eg. ($var * -1).
+            Tokens::$operators,
+            // Trying to compare a negative value; eg. ($var === -1).
+            Tokens::$comparisonTokens,
+            // Trying to compare a negative value; eg. ($var || -1 === $b).
+            Tokens::$booleanOperators,
+            // Trying to assign a negative value; eg. ($var = -1).
+            Tokens::$assignmentTokens,
+            [
```

Comment:
Please set this up with the tokens as the array keys, so you can use `isset()` instead of `in_array()` below.

Reply:
I don't think that this is necessary, but sure, I'll use `T_TOKEN => T_TOKEN` style, as used in the `Tokens` class.